var/spack/repos/builtin/packages/py-galaxy-tool-util/package.py

```diff
@@ -0,0 +1,99 @@
+    depends_on("py-galaxy-sequence-utils", type=("build", "run"))
+    depends_on("py-gravity@0.10.0:", type=("build", "run"))
+    depends_on("py-gunicorn", type=("build", "run"))
+    depends_on("py-gxformat2", type=("build", "run"))
+    depends_on("py-h5py", type=("build", "run"))
+    depends_on("py-importlib-resources", type=("build", "run"))
+    depends_on("py-isa-rwval", type=("build", "run"))
+    depends_on("py-kombu", type=("build", "run"))
+    depends_on("py-lagom", type=("build", "run"))
+    depends_on("py-mako", type=("build", "run"))
+    depends_on("py-markdown", type=("build", "run"))
+    depends_on("py-markupsafe", type=("build", "run"))
+    depends_on("mercurial", type=("build", "run"))
+    depends_on("py-mrcfile", type=("build", "run"))
+    depends_on("py-nodeenv", type=("build", "run"))
+    depends_on("py-nose", type=("build", "run"))
+    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-paramiko@:2.8.1,2.9.2:", type=("build", "run"))
+    depends_on("py-parsley", type=("build", "run"))
+    depends_on("py-paste", type=("build", "run"))
+    depends_on("py-pebble", type=("build", "run"))
+    depends_on("py-psutil", type=("build", "run"))
+    depends_on("py-pulsar-galaxy-lib@0.14.13:", type=("build", "run"))
+    depends_on("py-pycryptodome", type=("build", "run"))
+    depends_on("py-pydantic", type=("build", "run"))
+    depends_on("py-pyjwt", type=("build", "run"))
+    depends_on("py-pykwalify", type=("build", "run"))
+    depends_on("py-pyparsing", type=("build", "run"))
+    depends_on("py-pysam", type=("build", "run"))
+    depends_on("py-python-dateutil", type=("build", "run"))
+    depends_on("py-python-multipart", type=("build", "run"))
+    depends_on("py-pyuwsgi", type=("build", "run"))
+    depends_on("py-pyyaml", type=("build", "run"))
+    depends_on("py-refgenconf@0.12.0:", type=("build", "run"))
+    depends_on("py-requests", type=("build", "run"))
+    depends_on("py-routes", type=("build", "run"))
+    depends_on("py-social-auth-core@4.0.3+openidconnect", type=("build", "run"))
+    depends_on("py-sortedcontainers", type=("build", "run"))
+    depends_on("py-sqlalchemy@1.4.25:1", type=("build", "run"))
+    depends_on("py-sqlalchemy-migrate", type=("build", "run"))
+    depends_on("py-sqlitedict", type=("build", "run"))
+    depends_on("py-sqlparse", type=("build", "run"))
+    depends_on("py-starlette", type=("build", "run"))
+    depends_on("py-starlette-context", type=("build", "run"))
+    depends_on("py-svgwrite", type=("build", "run"))
+    depends_on("py-tifffile", type=("build", "run"))
+    depends_on("py-tuswsgi", type=("build", "run"))
+    depends_on("py-typing-extensions", type=("build", "run"))
+    depends_on("py-uvicorn", type=("build", "run"))
+    depends_on("py-uvloop", type=("build", "run"))
+    depends_on("py-webob", type=("build", "run"))
+    depends_on("py-whoosh", type=("build", "run"))
+    depends_on("py-zipstream-new", type=("build", "run"))
```

Comment:
22.1.5 only has the following dependencies:
```
galaxy-util>=22.1
galaxy-containers>=22.1
lxml
pydantic
pyyaml
sortedcontainers
typing-extensions
```

Reply:
done